src/directives/clickaway.js

```diff
@@ -39,20 +54,38 @@ function bind (el, binding, vnode) {
     //        the click, not whether it is there now, that the event has arrived
     //        to the top.
     // @NOTE: `.path` is non-standard, the standard way is `.composedPath()`
-    var path = ev.path || (ev.composedPath ? ev.composedPath() : undefined)
-    if (initialMacrotaskEnded && (path ? path.indexOf(el) < 0 : !el.contains(ev.target))) {
+    let panel = document.getElementById('alpheios-panel') ? document.getElementById('alpheios-panel').children[0] : null
```

Comment:
It would be more reliable to change selection logic here and/or below to ensure it will work in the case when some other element is inserted between a panel inner element and its outer container as in
```
<div id="alpheios-panel">
  <div id="something-new"></div>
  <div id="panel-component"></div>
</div>
```
and visibility of that element is different from the one of a container.
(as per discussion in Slack)